src/Application/Models/Asset.js

```diff
@@ -1,13 +1,15 @@
 import { assetDefaults } from './../../util/constants';
 
 class Asset {
-  constructor(name, price, multiplier, owned, unlocked) {
+  constructor(name, price, multiplier, owned, unlocked, investment, revenue) {
     this._name = name;
     this._price = price;
     this._multiplier = multiplier;
     this._owned = owned;
     this._unlocked = unlocked;
     this._defaults = assetDefaults[name];
+    this._investment = investment;
```

Comment:
also, consider applying the same scrutiny to commit messages.  "Add fields to Assets, Update Tests" gives almost no information without a lot of contextual understanding